sdk/mongocluster/Azure.ResourceManager.MongoCluster/README.md

```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure MongoCluster management client library for .NET
+
+**[Describe the service briefly first.]**
```

Comment:
I think we should update here.

Reply:
Don't think I have enough knowledge to update it here, will ping service team for this.